content/20.treatments.md

```diff
@@ -805,6 +805,17 @@ A list of current studies and their results, using carefully the information req
 **Summary.**
 Summarize the state of the neutralizing antibody approach.
 -->
+
+##### Adjuvants for Vaccines
+
+Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.
```

Comment:
I received another comment to say earlier what adjuvants are -- specifically, I can just take one of the later sentences and bring it to the beginning (while trying not to do it in a way which may cause a programming thing you guys mentioned earlier about moving stuff -- i'll just add and delete rather than cut/paste or move or something).